spring/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
We have global gitignore https://github.com/oskarv/spring-react/blob/master/.gitignore

Reply:
Good, we may need to update it once we have separate spring and react sub projects. I think it is a good idea to have separate gitignore for spring and react.